src/cli/customer.rs

```diff
@@ -26,7 +26,7 @@ pub struct Cli {
 #[derive(Debug, StructOpt)]
 pub enum Customer {
     List(List),
-    // Show(Show),
+    Show(Show),
```

Comment:
Why was it commented out?

Reply:
It wasn't implemented before for a single channel. I think it was just lower priority because we never checked a single channel at a time.